contracts/fixed_price_rev1.1.scilla

```diff
@@ -427,11 +452,19 @@ procedure HandleFulfillOrder(
   royalty_fee_bps <- & token_address.royalty_fee_bps;
   royalty_amount = portion sale_price royalty_fee_bps;
 
+  (* Calculating the Service Fee *)
   fee_bps <- service_fee_bps;
   svc_fee = portion sale_price fee_bps;
 
+  (* Calculating the optional Brand Commission Fee *)
+  opt_brand_commission_bps <- & collection_address.token_brand_fee[token_address][token_id];
```

Comment:
Can we include the logic for checking the brand commission recipient here? See my comment below.

Reply:
Outdated